govcd/vapp_test.go

```diff
@@ -143,7 +143,7 @@ func (vcd *TestVCD) Test_ChangeCPUCountWithCore(check *C) {
 		check.Skip("Skipping test because vApp was not successfully created at setup")
 	}
 
-	currentCpus := 0
+	currentCpus := int64(0)
```

Comment:
Wow, surprised we need to do this conversion here!

Reply:
Yep, we had to do it. Also in some places Assert compares not only values, but types as well and throws an error if it is not so.